src/assignments/assignment01.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) fn add(lhs: usize, rhs: usize) -> usize {
     todo!()
 }
 
-/// Adds two unsigned words. If underflow happens, just wrap around.
```

Comment:
- Adds -> Subtracts: thanks!
- underflow -> overflow: isn't it the case that subtraction may incur underflow rather than overflow?

Reply:
I thought this case is overflow, not underflow, according to this definition. https://en.wikipedia.org/wiki/Arithmetic_underflow